deblur/workflow.py

```diff
@@ -46,16 +52,19 @@ def dereplicate_seqs(seqs_fp,
     seqs_fp : string
         filepath to FASTA sequence file
     output_fp : string
-        filepath to dereplicated FASTA file
+        file path to dereplicated sequences (FASTA format)
     min_size : integer
         discard sequences with an abundance value smaller
         than integer
+    uc_output: boolean
```

Comment:
This parameter and the one above are missing the `optional`, as well as the description of the default.

```
min_size : integer, optional
    discard sequences with an abundance value smaller than integer. Default 2.
uc_output: boolean, optional
    output the dereplication map in .uc format. Default True.
```